roles/setup_assisted_installer/templates/configmap.yml.j2

```diff
@@ -32,3 +32,10 @@ data:
   INSTALLER_IMAGE: {{ assisted_service_installer_image }}
   CONTROLLER_IMAGE: {{ assisted_service_controller_image }}
   AGENT_DOCKER_IMAGE: {{ assisted_service_installer_agent_image }}
+  {% if http_proxy is defined %}
+  HTTP_PROXY: {{ http_proxy }}
+  {% elif https_proxy is defined %}
+  HTTPS_PROXY: {{ https_proxy }}
+  {% elif no_proxy is defined %}
+  NO_PROXY: {{ no_proxy }}
+  {% endif %}
```

Comment:
I'm pretty sure you want all 3 of these together so you don't want to use `elif`